python/nano/src/bigdl/nano/deps/ray/ray_distributed.py

```diff
@@ -156,6 +156,8 @@ def _wrapping_function(args_pack: tuple) -> Any:   # type: ignore[override]
         # patch Pytorch and CUDA in subprocess
         if os.environ.get('BIGDL_NANO_PATCH_TORCH') == '1':
             patch_cuda = True if os.environ.get('BIGDL_NANO_PATCH_CUDA') == '1' else False
+            os.environ['BIGDL_NANO_PATCH_TORCH'] = '0'
```

Comment:
Are there any places where we document the meaning of `BIGDL_NANO_PATCH_TORCH` and `BIGDL_NANO_PATCH_CUDA`? 
If there are not any, maybe we should add the document.

Reply:
They will only be used internally now, they are just used to indicate if patch has been called and whether subprocess should call patch. I'll eliminate their usage.